### PR TITLE
Store structured attributes from MongoDB logs in JSON format

### PR DESCRIPTION
**Pull request by jsoriano**
## What does this PR do?

Store the structured attributes included in JSON logs, using the `flattened` type for that.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Related issues

- JSON logs support added in #1138.

### CODE REVIEW

File: packages/mongodb/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.4.0"
+  changes:
+    - description: Store log attributes included in JSON logs
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1153
```
Comment (jsoriano) [inline packages/mongodb/changelog.yml:6]: @mtojek related to https://github.com/elastic/package-spec/pull/193, in this case I wanted to add this enhancement over the previous one, version 0.4.0 hasn't been released/promoted to any stage yet.

I could remove this entry as is an enhancement over the same unreleased feature introduced in the previous entry, but if I remove this entry, I would be missing the reference to one of the PRs.

Should I set the version of this entry to something like `0.4.1`, even if `0.4.0` never existed?

Comment (kaiyan-sheng) [reply packages/mongodb/changelog.yml:6]: Just a thought, if you want to keep the same version number `0.4.0`, could you add it under the previous entry like this?
```
- version: "0.4.0"
  changes:
    - description: Add support for logs in JSON format
      type: enhancement
      link: https://github.com/elastic/integrations/pull/1138
    - description: Store log attributes included in JSON logs
      type: enhancement
      link: https://github.com/elastic/integrations/pull/1153
```

Comment (mtojek) [reply packages/mongodb/changelog.yml:6]: Kaiyan is absolutely right here.

> Should I set the version of this entry to something like 0.4.1, even if 0.4.0 never existed?

With the mentioned PR, there will be officially validated other option we had in mind: `0.4.1-next` (unreleased yet). The package manifest can be left as is.

Comment (jsoriano) [reply packages/mongodb/changelog.yml:6]: Oh ok, makes totally sense, I didn't think in `changes` as a list :+1:

**Review by mtojek — APPROVED** [review]

**Pull request merged**